.github/workflows/ci.yml

```diff
@@ -504,23 +504,24 @@ jobs:
         run: pytest --no-sqlalchemy --aws-integration -v tests/integration/usage_statistics
 
   build-n-publish:
-    needs: [
-      ci-is-on-main-repo,
-      doc-checks,
-      static-analysis,
-      docs-snippets,
-      unit-tests,
-      cloud-tests,
-      marker-tests,
-      py38-min-versions,
-      py39-min-versions,
-      py310-min-versions,
-      py311-min-versions,
-      pydantic-v1,
-      airflow220-min-versions,
-      import_gx,
-      usage_stats_integration
-    ]
+    needs:
```

Comment:
Why are these lines changing? Can you install the pre-commit hooks as they will prevent whitespace changes from accidentally being committed